integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -1288,4 +1289,87 @@ protected JSONObject executeQuery(String query) throws IOException {
+    verifySchema(result, schema("{ts '2020-09-16 17:30:00'}", null, "timestamp"));
+    verifyDataRows(result, rows("2020-09-16 17:30:00"));
+
+    result = executeQuery("select {timestamp '2020-09-16 17:30:00.123'}");
+    verifySchema(result, schema("{timestamp '2020-09-16 17:30:00.123'}", null, "timestamp"));
+    verifyDataRows(result, rows("2020-09-16 17:30:00.123"));
+
+    result = executeQuery("select {ts '2020-09-16 17:30:00.123'}");
+    verifySchema(result, schema("{ts '2020-09-16 17:30:00.123'}", null, "timestamp"));
+    verifyDataRows(result, rows("2020-09-16 17:30:00.123"));
+  }
+
+  @Test
+  public void testTimeBracket() throws IOException {
+    JSONObject result = executeQuery("select {time '17:30:00'}");
+    verifySchema(result, schema("{time '17:30:00'}", null, "time"));
+    verifyDataRows(result, rows("17:30:00"));
+
+    result = executeQuery("select {t '17:30:00'}");
+    verifySchema(result, schema("{t '17:30:00'}", null, "time"));
+    verifyDataRows(result, rows("17:30:00"));
+
+    result = executeQuery("select {time '17:30:00'}");
+    verifySchema(result, schema("{time '17:30:00'}", null, "time"));
+    verifyDataRows(result, rows("17:30:00"));
+
+    result = executeQuery("select {t '17:30:00'}");
+    verifySchema(result, schema("{t '17:30:00'}", null, "time"));
+    verifyDataRows(result, rows("17:30:00"));
+  }
+
+  @Test
+  public void testDateBracket() throws IOException {
+    JSONObject result = executeQuery("select {date '2020-09-16'}");
+    verifySchema(result, schema("{date '2020-09-16'}", null, "date"));
+    verifyDataRows(result, rows("2020-09-16"));
+
+    result = executeQuery("select {d '2020-09-16'}");
+    verifySchema(result, schema("{d '2020-09-16'}", null, "date"));
+    verifyDataRows(result, rows("2020-09-16"));
+  }
+
+  private void compareBrackets(String query1, String query2, String datetime) throws IOException {
+    JSONObject result1 = executeQuery("select " + query1 + " '" + datetime + "'");
+    JSONObject result2 = executeQuery("select {" + query2 + " '" + datetime + "'}");
+
+    verifyDataRows(result1, rows(datetime));
+    verifyDataRows(result2, rows(datetime));
+  }
+
+  @Test
+  public void testBracketedEquivalent() throws IOException {
+    compareBrackets("timestamp", "timestamp", "2020-09-16 17:30:00");
+    compareBrackets("timestamp", "ts", "2020-09-16 17:30:00");
+    compareBrackets("timestamp", "timestamp", "2020-09-16 17:30:00.123");
+    compareBrackets("timestamp", "ts", "2020-09-16 17:30:00.123");
+    compareBrackets("date", "date", "2020-09-16");
+    compareBrackets("date", "d", "2020-09-16");
+    compareBrackets("time", "time", "17:30:00");
+    compareBrackets("time", "t", "17:30:00");
+  }
+
+  private void queryFails(String query) {
+    assertThrows(ResponseException.class, ()->executeQuery(query));
+  }
+
+  @Test
+  public void testBracketFails() {
+    queryFails("select {time 'failure'}");
```

Comment:
Since `queryFails` only does one thing, we can just have `assertThrows(ResponseException.class, ()-> executeQuery(query));` on every line in this test case.

Reply:
That's a good point, changed in https://github.com/opensearch-project/sql/pull/1894/commits/3ba8b5ecd289e506f527deef538219b286a3d8d7